app/components/paperShow/components/plutoBlogPosting.tsx

```diff
@@ -1,31 +1,25 @@
 import * as React from "react";
 import axios from "axios";
 import { withStyles } from "../../../helpers/withStylesHelper";
+import { BlogLink } from "../../../containers/admin";
 const styles = require("./plutoBlogPosting.scss");
 
 const BLOG_SCRIBER_API_HOST = "https://7hnqfzk1r6.execute-api.us-east-1.amazonaws.com/prod/blogLinks";
 
-interface BlogLink {
-  id: string;
-  link: string;
-  active: boolean;
-  startTime?: Date[];
-  endTime?: Date[];
-  createdAt?: Date;
-  updatedAt?: Date;
-  ogImageUrl?: string;
-  ogTitle?: string;
-  ogDescription?: string;
-}
+interface BlogInfo extends BlogLink {}
 
 interface PlutoBlogPostingState {
   isLoading: boolean;
   blogLink: string;
-  blogLinks: BlogLink[];
+  blogLinks: BlogInfo[];
+}
+
+interface PlutoBlogPostingProps {
+  paperId: String;
```

Comment:
You should use `number` here.
(or `string`(lowercase). Typescript interface doesn't work well with `String`.